### ISSUE TITLE
Update request: mpv 0.36.0 → 0.37.0

### ISSUE THREAD

**Issue by Tungsten842** [NixOS/nixpkgs#270002]
https://github.com/mpv-player/mpv/releases/tag/v0.37.0
- Package name: mpv
- Latest released version: 0.37.0

- Current version on the unstable channel: 0.36.0
- Current version on the stable/release channel: 0.35.1

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**
@AndersonTorres @fpletz @globin @Ma27 @tadeokondrak 

-----

Note for maintainers: Please tag this issue in your PR.

**Priorities**

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by arthsmn** [comment]
The new mpv version relies on libplacebo 6.338.1, which is [being worked on in this PR](https://github.com/NixOS/nixpkgs/pull/269415).

**Comment by rhododendrox** [comment]
In case someone doesn't want to wait for the official update here's the overlay I'm currently using to build mpv 0.37.0.

```nix
final: prev:

{

  mpv-unwrapped =
    let
      newLibplacebo = prev.pkgs.libplacebo.overrideAttrs (oa: rec {
          version = "6.338.1";

          src = prev.pkgs.fetchFromGitLab {
            domain = "code.videolan.org";
            owner = "videolan";
            repo = oa.pname;
            rev = "v${version}";
            hash = "sha256-NZmwR3+lIC2PF+k+kqCjoMYkMM/PKOJmDwAq7t6YONY=";
          };

          buildInputs = oa.buildInputs ++ [ prev.pkgs.xxHash ];
      });
    in
      (prev.mpv-unwrapped.override { libplacebo = newLibplacebo; }).overrideAttrs (oa: rec {
        version = "0.37.0";

        src = prev.pkgs.fetchFromGitHub {
          owner = "mpv-player";
          repo = "mpv";
          rev = "v${version}";
          hash = "sha256-izAz9Iiam7tJAWIQkmn2cKOfoaog8oPKq4sOUtp1nvU=";
        };
      });

}
```

**Comment by arthsmn** [comment]
Just as an update. #271905 was merged, now it's only a matter of #269415 being merged and then the update can occur.
